packages/wonder-blocks-dropdown/components/action-menu.md

```diff
@@ -267,4 +267,41 @@ class ControlledActionMenuExample extends React.Component {
 }
 
 <ControlledActionMenuExample />
-```
\ No newline at end of file
+```
+
+### Example: ActionMenu with custom opener
+
+In case you need to use a custom opener, you can use the `opener` property to achieve this:
+
+```js
+import {ActionMenu, ActionItem, OptionItem, SeparatorItem} from "@khanacademy/wonder-blocks-dropdown";
+import {View} from "@khanacademy/wonder-blocks-core";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
+
+<ActionMenu
+    disabled={false}
+    menuText="Betsy Appleseed"
+    testId="teacher-menu"
+    opener={(eventState) => (
+        <LabelMedium
+            onClick={()=>{console.log('custom click!!!!!')}}
+        >This is a label</LabelMedium>
+    )}
+>
+    <ActionItem label="Profile" href="http://khanacademy.org/profile" testId="profile" />
+    <ActionItem label="Settings" onClick={() => console.log("user clicked on settings")} testId="settings" />
+    <ActionItem label="Help" disabled={true} onClick={() => console.log("this item is disabled...")} testId="help" />
+    <SeparatorItem />
+    <ActionItem label="Log out" href="http://khanacademy.org/logout" testId="logout" />
+    <OptionItem
+        label="Show homework assignments" value="homework"
+        onClick={() => console.log(`Show homework assignments toggled`)}
+    />
+    <OptionItem
+        label="Show in-class assignments" value="in-class"
+        onClick={() => console.log(`Show in-class assignments toggled`)}
+    />
```

Comment:
Is there a way to adjust the position of the dropdown relative to the opener?

Reply:
Not sure if I follow, do you mean something like adding `padding` or `margin` to the opener?

Reply:
Or on the dropdown itself since vertical padding/margin might affect the position of the opener itself.